docs/release-notes.rst

```diff
@@ -1,3 +1,277 @@
+
+**Workaround:** Use a different browser than IE.
+
+------
+
+**BIG-IP version may not be displayed correctly in FASC when BIG-IP version is upgraded or downgraded.**
+
+The BIG-IP version displayed on hover over the BIG-IP IP address may not be displayed correctly when BIG-IP is upgraded or downgraded.
+
+**Workaround:** Log-out and re-login the BIG-IP device in the FASC application.
+
+------
+
+**For non-admin BIG-IP users, Default Gateway CRUD operations show success even though the corresponding operation on the BIG-IP is not permitted and failed.**
+
+Non-admin BIG-IP users have restricted access to F5 ACI ServiceCenter, however due to an Ansible defect, users might see success messages for Default Gateway operations despite those operations failing on the BIG-IP. 
+
+**Ansible Defect:** https://github.com/F5Networks/f5-ansible/issues/2088
+
+**Workaround:** Use BIG-IP admin account for F5 ACI ServiceCenter Default Gateway operations.
+
+------
+
+**BIG-IP LDAP User with role "No-Access" can login to F5 ACI ServiceCenter with limited access**
+
+Access to the F5 ACI ServiceCenter operations for LDAP users depends upon whether the user is admin or non-admin. F5 ACI ServiceCenter is not able to distinguish between non-admin access roles such as operator, guest, no access and so on, hence users with role **No-Access** might be able to login to F5 ACI ServiceCenter with limited access.
+
+**Workaround:** Use BIG-IP admin account for F5 ACI ServiceCenter operations. 
+
+------
+
+
+Visibility
+----------
+
+**Visibility report downloads do not work in the Chrome browser version 83 and higher (Fixed in APIC 4.2(4p)+ and 5.0(2e)+)**
+
+F5 ACI ServiceCenter's Visibility tab has a **Download Report** icon for downloading VLAN, VIP and Node table data in CSV format. This download functionality does not work in the Chrome browser version 83 and higher, due to a new security check added by Chrome: https://developers.google.com/web/updates/2020/04/chrome-83-deps-rems#disallow_downloads_in_sandboxed_iframes 
+
+**Fix:** This has been fixed in APIC versions 4.2(4p)+ and 5.0(2e)+.
+
+**Workaround:** To download visibility reports, use a different browser, like Mozilla Firefox, or an earlier version of Google Chrome.
+
+------
+
+**Visibility table fast scrolling on Mozilla Firefox for scale configs may result in a blank screen**
+
+This is a known ag-grid issue on the Mozilla Firefox browser: https://github.com/ag-grid/ag-grid/issues/2841
+
+**Workaround:** Scroll slowly to prevent this issue. But if this issue is observed, the possible workarounds are:
+
+1. Switch tabs from the Visibility tab to one of the other tabs and then switch back. 
+
+2. Select a different entry from the Visibility table drop-downs (either Partition or Table) and switch back to the intended combination.
+
+3. Close and re-open the FASC app UI.
+
+------
+
+**Visibility table status icons render few seconds after data for scale configs.**
```

Comment:
(just noticed the period at the end of some of these headings.  Either with or without period is probably fine, but be consistent (ie either remove or add all periods at the end)